app/views/static_pages/about.html.erb

```diff
@@ -0,0 +1,20 @@
+<% provide :title, "About" %>
```

Comment:
I18n cho `About`

Và dingf i18n cho tất cả các text của html file

Search và tìm hiểu key I18n in rails hì